### ISSUE TITLE
Add rendering for information=stele

### ISSUE THREAD

**Issue by AragonChristopherR17Z** [gravitystorm/openstreetmap-carto#4932]
### Expected behavior
Other similar [information= ](https://wiki.openstreetmap.org/wiki/Key:information)objects such as information=board, information=terminal, information=map, and information=guidepost are rendered in the map.
### Actual behavior
Currently, [information=stele](https://wiki.openstreetmap.org/wiki/Tag:information%3Dstele) is not rendered in the map, despite having 1510 uses, most of which (1474) are nodes, according to [taginfo](https://taginfo.openstreetmap.org/tags/information=stele#overview). This _is_ lower than the other information= objects; however, this could be because it does not appear as a suggestion when searching for tags to add to nodes on OSM as well as the fact that it is not rendered.
This feature is also rendered by other maps as noted in the [talk page for the tag](https://wiki.openstreetmap.org/wiki/Talk:Tag:information%3Dstele).
### Links illustrating the problem
Here I'll provide a link to a node that is not rendered in OSM which was used as an example by Pyrog in the Wiki Talk page.
[Example.](https://www.openstreetmap.org/node/10748194327/)
![Stele not rendered](https://github.com/gravitystorm/openstreetmap-carto/assets/10082060/13fa9e2c-a802-4de4-b25a-a719e8747135)
VS
![Rendered in Tracestrack Topo](https://github.com/gravitystorm/openstreetmap-carto/assets/10082060/1dffe153-3f3f-4a58-906a-409e06d5bc24)

**Comment by imagico** [comment]
Thanks for the suggestion.

Tag has ~1500 uses, 1100 of this in Russia, 200 in Germany, 200 in the rest of the world.  Documentation on the wiki is very sparse and self referential (i.e. language specific) with no meaningful delineation towards other tags.

I am not opposed to rendering this in principle, but so far there is not even enough information on what mappers who invented the tag intend it to be used for to check if it is consistently used for that purpose.  Global distribution indicates there are likely different regional practices here.

> This feature is also rendered by other maps as noted in the [talk page for the tag](https://wiki.openstreetmap.org/wiki/Talk:Tag:information%3Dstele).

I am pretty sure most of these maps render generically `tourism=information`, not specifically `information=stele`.  We deliberately stopped rendering `tourism=information` universally some time ago (#4420).

**Comment by AragonChristopherR17Z** [comment]
> Tag has ~1500 uses, 1100 of this in Russia, 200 in Germany, 200 in the rest of the world. Documentation on the wiki is very sparse and self referential (i.e. language specific) with no meaningful delineation towards other tags.
> 
> I am not opposed to rendering this in principle, but so far there is not even enough information on what mappers who invented the tag intend it to be used for to check if it is consistently used for that purpose. Global distribution indicates there are likely different regional practices here.
The Wiki description is pretty simplistic and does allow for a vague interpretation. I went ahead and looked for different instances of this tag and noticed some interesting things: In the instances I found in Russia, the tag was used for monument-like structures, usually but [not always](https://www.openstreetmap.org/way/837670005) taller than they are wide, and were usually a kind of [welcome monument](https://www.openstreetmap.org/node/4441723611) with a localities' name.
Outside of Russia, the tag is generally used for what is depicted in the Wiki, a tall board with information about a place, directions to other amenities, and sometimes maps. This was generally the case of what I found in Germany, Poland, the UK, and France, with few exceptions where it is used on [stones with inscriptions](https://www.openstreetmap.org/node/11087165870),[ information plaques](https://www.openstreetmap.org/node/9452558500) on stone, and what I think is actually a [water level measuring stick](https://www.openstreetmap.org/node/5969236629).
In Japan, where there are only a handful of instances, this tag was used for a tall [information post](https://www.openstreetmap.org/node/10116715713), but also for a [large stone block](https://www.openstreetmap.org/node/10628469099) with etched inscriptions and in [one case](https://www.openstreetmap.org/node/9121566748) what would more accurately be an information board.
In the US, the use is generally the same as in central Europe, with exceptions being [cases ](https://www.openstreetmap.org/node/11376121485)where an information board tag may or may not be more appropriate as well as some cases similar to Russia where it was used for a [locality sign](https://www.openstreetmap.org/node/6719975170).
In South America, there were cases where it was used for tall welcome signs to a locality and a park.

Basically, the problem with the tag is that there's overlap with welcome signs, regular information boards, and (probably because a stele implies a stone structure) plaques or inscriptions on stones; but, they are still generally used, except for in Russia apparently, to denote a tall structure which functions as a type of information board with place names, directions, amenities, maps, etc. There should be an effort to differentiate it from welcome signs, monuments, and plaques which would help to give this tag a more concrete meaning/purpose.
